libs/blocks/accordion/accordion.css

```diff
@@ -144,3 +144,82 @@ dl.accordion {
 .dark .accordion dt button:hover .accordion-icon::after {
   background: var(--color-gray-100);
 }
+
+/* Editorial Variation  */
+.accordion-media {
+  display:  none;
+}
+
+.accordion-media > div {
+  position: relative;
+  display: none;
+  animation-duration: .1s;
+  animation-name: slide-left;
+}
+
+/***** Slide to the Left *****/
+.slide-left {
+  animation: 1s slide-left;
+}
+@keyframes slide-left {
+  from {
+    margin-right: -100%;
+  }
+  to {
+    margin-right: 0%;
+  }
+}
+
+.accordion-media > div.expanded, 
+.accordion-media > div.expanded > img {
+  display: inline;
+  position: relative;
+  max-height: 525px;
+  width: auto;
+  max-width: 700px;
+  animation-duration: .1s;
```

Comment:
If you do like the fade better I'd bump this up to around .3-.5 for the fade time.

It also appears this is the only duration you need.